### ISSUE TITLE
potentially unwanted behaviour: idientifier 'have' is alias for 'must'

### ISSUE THREAD

**Issue by jages** [openvalidation/openvalidation#72]
## Expected Behavior & Current Behaviour

following test rule
```
if your name is peter
    and your location is Berlin
thne you have a problem
```
with following YAML shema:
```yaml
name: Satoshi
age: 25
location: Dortmund
```
because of the typo in `thne`, this should not be recognized as a rule, but `have` is detected as an alias for `must` (akin to `X has to be Y`), therefore this rule compiles to a somewhat unexpected must construct.

## Possible Solution

we might want to re-think, if i want to allow more freedom in expressing errors in rules, or if we double down in strictly enforcing `if A then error` syntax

## Steps to Reproduce (for bugs)
Copy & paste rule and schema into playground.openvalidation.io , or use the above and paste it into CLI

**Comment by jages** [comment]
This behaviour is correct, but might do with a better error report.
